view/frontend/web/js/model/express-pay.js

```diff
@@ -50,14 +61,14 @@ define([
             if (testMode) {
                 parameters = '&debug=true';
             }
-            if (!require.defined('bold_paypal_sdk')){
+            if (!require.defined('bold_paypal_fastlane')){
```

Comment:
Put this back to `bold_paypal_fastlane` as once we switch over to using the SPI this logic will likely be removed.